src/ButtonToggle.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import Button from "./Button";
+import classNames from 'classnames';
+import { mapToCssModules } from './utils';
+
+const propTypes = {
+  onClick: PropTypes.func,
+  onBlur: PropTypes.func,
+  onFocus: PropTypes.func,
+  disabled: PropTypes.bool,
+}
+
+class ButtonToggle extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      toggle: false,
```

Comment:
I'll use the `defaultProps` property for this like in `Button.js` file.
I'll use the name `toggleValue` instead of `defaultValue`. Cool?
Also, Should I do the same for `focus` state as well like `props.focusValue` which can be passed down from parent?

Reply:
defaultProps is fine.
I don't think the focus should be since it will not actually _be_ focused (and thus cannot be blurred to trigger the non-focus styled)